specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/AppSpaceResources.json

```diff
@@ -0,0 +1,433 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-11-02-preview",
+    "title": "ContainerApps API Client"
+  },
```

Comment:
Can you please fix the newly flagged errors AND warnings from the Lint diff here : 1

[https://github.com/Azure/azure-rest-api-specs/pull/27483/checks?check_run_id=20787124422](https://github.com/Azure/azure-rest-api-specs/pull/27483/checks?check_run_id=20787124422)

Reply:
Hello! I could use some help - I was using previous examples in our manifest to make these new resource types and the errors seem to be in PATCH because for tracked resources, the parameter in the common v3 tracked resource is a required param, which is the error in lintdiff for patch method. What do you recommend to fix this issue?

Reply:
Resolved all the lintdiff errors and warnings.